app/src/main/java/com/keylesspalace/tusky/components/preference/ProxyPreferencesFragment.kt

```diff
@@ -16,38 +16,58 @@
 package com.keylesspalace.tusky.components.preference
 
 import android.os.Bundle
+import androidx.preference.Preference
 import androidx.preference.PreferenceFragmentCompat
 import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.entity.ProxyConfiguration
 import com.keylesspalace.tusky.settings.PrefKeys
-import com.keylesspalace.tusky.settings.editTextPreference
 import com.keylesspalace.tusky.settings.makePreferenceScreen
 import com.keylesspalace.tusky.settings.switchPreference
+import com.keylesspalace.tusky.settings.validatedEditTextPreference
 import kotlin.system.exitProcess
 
 class ProxyPreferencesFragment : PreferenceFragmentCompat() {
     private var pendingRestart = false
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         makePreferenceScreen {
-            switchPreference {
+            val enableToggle = switchPreference {
                 setTitle(R.string.pref_title_http_proxy_enable)
                 isIconSpaceReserved = false
                 key = PrefKeys.HTTP_PROXY_ENABLED
                 setDefaultValue(false)
             }
 
-            editTextPreference {
```

Comment:
`editTextPreference` is now unused and can be deleted.

Reply:
Gotcha. I was afraid to be too aggressive with deleting unused code I didn't write!